var/spack/repos/builtin/packages/py-pytng/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPytng(PythonPackage):
+    """Provides the TNGFileIterator object to allow simple Pythonic
+    access to data contained within TNG files.
+    """
+
+    homepage = "https://pypi.org/project/pytng/"
+    pypi = "pytng/pytng-0.3.0.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.3.0", sha256="f563f9ea260ca8c8e17b3bcf9458bae35aedd5c58e1c5ac4dfff77a1e036506e")
+
+    depends_on("python@3.8:")
+
+    depends_on("py-cython@0.28:0.29.33", type="build")
+    depends_on("py-setuptools@42:", type="build")
```

Comment:
Where do you see this version requirement?

Reply:
```suggestion
    depends_on("py-setuptools", type="build")
```